Packs/CrowdStrikeFalconSandbox/Integrations/CrowdstrikeFalconSandboxV2/README.md

```diff
@@ -0,0 +1,4459 @@
+| Polling | . Possible values are: true, false. | Optional | 
+| JobID | JobID to retrieve scan results for. | Optional | 
+| hide_polling_output | . | Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| CrowdStrike.Report.job_id | Unknown |  | 
+| CrowdStrike.Report.environment_id | Unknown |  | 
+| CrowdStrike.Report.environment_description | String |  | 
+| CrowdStrike.Report.size | Number |  | 
+| CrowdStrike.Report.type | String |  | 
+| CrowdStrike.Report.type_short | String |  | 
+| CrowdStrike.Report.target_url | Unknown |  | 
+| CrowdStrike.Report.state | String |  | 
+| CrowdStrike.Report.error_type | Unknown |  | 
+| CrowdStrike.Report.error_origin | Unknown |  | 
+| CrowdStrike.Report.submit_name | String |  | 
+| CrowdStrike.Report.md5 | String |  | 
+| CrowdStrike.Report.sha1 | String |  | 
+| CrowdStrike.Report.sha256 | String |  | 
+| CrowdStrike.Report.sha512 | String |  | 
+| CrowdStrike.Report.ssdeep | Unknown |  | 
+| CrowdStrike.Report.imphash | Unknown |  | 
+| CrowdStrike.Report.av_detect | Number |  | 
+| CrowdStrike.Report.vx_family | String |  | 
+| CrowdStrike.Report.url_analysis | Boolean |  | 
+| CrowdStrike.Report.analysis_start_time | Date |  | 
+| CrowdStrike.Report.threat_score | Unknown |  | 
+| CrowdStrike.Report.interesting | Boolean |  | 
+| CrowdStrike.Report.threat_level | Number |  | 
+| CrowdStrike.Report.verdict | String |  | 
+| CrowdStrike.Report.total_network_connections | Number |  | 
+| CrowdStrike.Report.total_processes | Number |  | 
+| CrowdStrike.Report.total_signatures | Number |  | 
+| CrowdStrike.Report.file_metadata | Unknown |  | 
+| CrowdStrike.Report.submissions.submission_id | String |  | 
+| CrowdStrike.Report.submissions.filename | String |  | 
+| CrowdStrike.Report.submissions.url | Unknown |  | 
+| CrowdStrike.Report.submissions.created_at | Date |  | 
+| CrowdStrike.Report.network_mode | String |  | 
+| File.SHA256 | string | SHA256 of the file | 
+| File.SHA1 | string | SHA1 of the file | 
+| File.MD5 | string | MD5 of the file | 
+| File.Name | string | Submission name of the file | 
+| File.MalwareFamily | string | Family classification of the file | 
+| File.Malicious.Vendor | string | or malicious files, the vendor that made the decision | 
+| File.Malicious.Description | string | For malicious files, the reason for the vendor to make the decision | 
+| DBotScore.Indicator | string | The indicator we tested | 
+| DBotScore.Type | string | The type of the indicator | 
+| DBotScore.Vendor | string | Vendor used to calculate the score | 
+| DBotScore.Score | number | The actual score | 
+
+#### Command example
+```!cs-falcon-sandbox-scan file=8decc8571946d4cd70a024949e033a2a2a54377fe9f1c1b944c20f9ee11a9e51,9745bd652c50ac081e28981b96f41230c1ed2f84724c1e5b0f0d407a90aefe22```
+#### Context Example
+```json
+{
+    "CrowdStrike": {
```

Comment:
please see if we can make the res here more lean, we dont really need the whole object

Reply:
just in the docs? Or also in the yaml?

Reply:
just in the docs